VMEncryption/main/HttpUtil.py

```diff
@@ -19,18 +19,19 @@
 import time
 import datetime
 import traceback
-import urlparse
-import httplib
+import urllib.parse
+import http.client
 import shlex
 import subprocess
 from Common import CommonVariables
 from subprocess import *
-from Utils.WAAgentUtil import waagent
+from Utils.waagentloader import load_waagent
 
 class HttpUtil(object):
     """description of class"""
     def __init__(self, logger):
         self.logger = logger
+        waagent = load_waagent()
```

Comment:
comment that we are loading our own version of python3 compatible waagent

Reply:
check if waagent python3 is compatible with python2

Reply:
comment added, dual compatibility same as vmaccess, any issues will be exposed during testing